hsbc/pyratings
# docs: Add CONTRIBUTING.md and CODE_OF_CONDUCT.md

- Added CONTRIBUTING.md document that should help people to get in touch with us and facilitate contributing to the project.
- Added CODE_OF_CONDUCT.md.

Please check both documents! You could read them directly in GitHub by switching to the ``contributing`` branch.

Having said that, it would also be good to check if the incorporation into the online documentation looks like expected.

Follow these steps to check the docs:
- Activate your virtual environment
- Pull ``contributing`` branch:
  - ``git fetch``
  - ``git checkout -t origin/contributing``
- ``pip install --upgrade -e .[dev,docs,tests]``
- ``mkdocs serve``

You can then check the documentation at http://localhost:8000.
